docs/config/ping-federate-example.md

```diff
@@ -9,23 +9,29 @@ Ensure you have installed the Identity Service before starting. You will also ne
 ## Configuration
 There are three main steps involved in configuring a PingFederate instance with the Identity Service:
 
-1. Obtain your parameters from PingFederate.
-2. Configure the Identity Service with your PingFederate parameters.
-3. Configure your PingFederate connection.
+1. Get the configuration values. 
+2. Configure your PingFederate connection.
+3. Configure the Identity Service with your PingFederate parameters.
 
-### 1. Obtain parameters from PingFederate
-1. Sign in to your PingFederate instance as a user with administrative privileges.
-2. Select **Server Settings** from the **System Settings** menu.
-3. Make a note of the value for *My Base URL* under the **Federation Info** heading.
-4. Click **Main** and select **Federation Settings** or **IdP Configuration**, then click **Protocol Endpoints**.
-5. Make a note of the *Redirect* values for *Single Logout (SLO) Service* and *Single Sign-on (SSO) Service* under the **SAML v2.0 Endpoints** heading.
-6. Combine the value of *My Base URL* (from Step 3) with *Single Logout (SLO) Service* and *My Base URL* (from Step 3) with *Single Sign-on (SSO) Service* ready to enter into the Identity Service.
+### Get the configuration values. 
 
-For example:
-> If *My Base URL* is `"https://pingfederate.test.com:9031"` and *Single Sign Logout (SLO) Service* is `"/idp/SLO.saml2"` then the POST SLO service URL is `https://pingfederate.test.com:9031/idp/SLO.saml2`
+1. Open a new text file and paste the following into it.
+```
+-----BEGIN CERTIFICATE-----
 
-### 2. Configure the Identity Service
-1. Sign in to the administrator panel of the Identity Service using the following URL: `https://$ELBADDRESS/auth/admin`
+-----END CERTIFICATE-----
+```
+2. In a separate tab, open the **certificate descriptor API** at **$ELBADDRESS/auth/realms/alfresco/protocol/saml/descriptor**.
+3. In the middle of the two lines, add the value found between **<dsig:X509Certificate>** and **</dsig:X509Certificate>** in the **certificate descriptor API** such as in the following example.
```

Comment:
Again, unclear. Perhaps, instead of switching back and forth between an editor and a Web browser, a user could 1st go open **http://$ELBADDRESS/auth/realms/alfresco/protocol/saml/descriptor**, copy the value in-between these XML tags and then move on to the text file?